lib/components/form/advanced-settings-panel.tsx

```diff
@@ -28,20 +20,30 @@ import PageTitle from '../util/page-title'
 
 import React, { RefObject, useContext } from 'react'
 
+import * as apiActions from '../../actions/api'
+import * as formActions from '../../actions/form'
+
 import {
   addCustomSettingLabels,
   addModeButtonIcon,
+  alertUserTripPlan,
   onSettingsUpdate,
   pipe,
   populateSettingWithIcon,
   setModeButton
 } from './util'
+import { AppReduxState } from '../../util/state-types'
 import { setModeButtonEnabled } from './batch-settings'
 
+import { blue, getBaseColor } from '../util/colors'
+import { ComponentContext } from '../../util/contexts'
+
 import styled from 'styled-components'
 
+const baseColor = getBaseColor()
```

Comment:
not working for me. potential timing issues with when the function is called and css is loaded?

Reply:
You were right, the CSS variable isn't loaded yet when this got called. I moved it around and used the vanilla CSS variable in the styled components, no need to take a CSS variable, make it a JS variable, just to put it back in the CSS!